cli/tools/jupyter/server.rs

```diff
@@ -500,7 +498,10 @@ async fn get_jupyter_display(
       return Ok(Some(data));
     }
   } else {
-    eprintln!("Unexpected response from Jupyter.display: {:?}", json_str);
+    eprintln!(
+      "Unexpected response from Jupyter.display: {:?}",
+      response.result.clone().value.unwrap_or_default()
```

Comment:
Skip the `unwrap_or_default` call here as the "debug"-style print (ie: `:?`) will show us the `None`.